functions/update-many/1.0/index.js

```diff
@@ -0,0 +1,37 @@
+import { parseAssignedProperties, fetchRecords } from '../../utils';
+
+const updateMany = async ({
+  selectedCollection: {
+    data,
+    model: { name: modelName },
+  },
+  mapping,
+}) => {
+  const mutationName = `updateMany${modelName}`;
+  const assignProperties = parseAssignedProperties(mapping);
+  const ids = data.map((item) => item.id);
+  const mutation = `
+    mutation($input: ${modelName}Input, $where: Many${modelName}FilterInput) {
+      ${mutationName}(input: $input, where: $where) {
+        id
+      }
+    }
+  `;
+
+  const { errors } = await gql(mutation, {
+    input: assignProperties,
+    where: { id: { in: ids } },
+  });
+
+  if (errors) {
+    throw errors;
+  }
+
+  const updatedRecords = await fetchRecords(modelName, ids, mapping);
```

Comment:
Is the purpose of this function to also fetch the records? I personally think that would be another step. Since its not always nescarry but its always executed

Reply:
That is true, it won't be necessary in any case. However, I have tried to conditionally execute this statement based on whether the output variable is filled in, but we can't do that. The output variable is not passed to the function.

Reply:
Then please remove it, it causes degraded performance and makes to many requests to Data-Api